### DISCUSSION TITLE
Can I hide the input (id_location) box but retain the others?

### DISCUSSION THREAD

**Discussion by tim-blackmore** [caioariede/django-location-field#165] [General]
Hopefully a simple one, but I am struggling to hide one of the input boxes. Does anyone have any advice?

![image](https://user-images.githubusercontent.com/24419429/98439542-71e1ed80-20ea-11eb-9028-c0ab64a3309e.png)

**Comment by iragm** [comment]
Not the cleanest, but assuming your field is called `location`, add to your template:

```
<script>
    $( document ).ready(function() {
        document.getElementById('id_location').style.display = "none";
    });
</script>
```

**Comment by tim-blackmore** [comment]
Thank you!

**Comment by aniruddha-ca** [comment]
Thanks @iragm . But when I use JavaScript or CSS to hide id_location, even the map is not visible. @tim-blackmore

**Comment by iragm** [comment]
Make sure you are not hiding the whole div -- `div_id_location_coordinates` will hide everything.  The above code works for me.  Post code from your model and template.

**Comment by humpfreybogart** [comment]
or add something like this in your css file

`form input#id_location {display:none;}`

replace "id_location" with the input id off course

want to hide the label text then add something lik this to your css file

`form label[for="id_location"] {display:none;}`
